web-client/src/presenter/sequences/displayProgressSpinnerSequence.ts

```diff
@@ -0,0 +1,8 @@
+import { showProgressSequenceDecorator } from '@web-client/presenter/utilities/showProgressSequenceDecorator';
+
+export const displayProgressSpinnerSequence = showProgressSequenceDecorator([
+  async ({ props }: ActionProps<{ timeInSeconds: number }>) =>
+    await new Promise(resolve =>
+      setTimeout(() => resolve(null), props.timeInSeconds * 1000),
+    ),
```

Comment:
Why is this necessary?

Reply:
I _think_ the idea is that when filtering is fast, it looks like nothing changed, so this is just a signal to the user. I asked a similar question.

Reply:
Thank you! That's a good problem to have

Reply:
That is correct, mainly in mobile UX team stated that the filters would take most of screen and so users will not see changes in the table. So this is to provide a UI indication to the user that something has happened once they change some filter.